scene/main/http_request.cpp

```diff
@@ -81,7 +81,7 @@ Error HTTPRequest::request_raw(const String &p_url, const Vector<String> &p_cust
 	ERR_FAIL_COND_V(!is_inside_tree(), ERR_UNCONFIGURED);
 	ERR_FAIL_COND_V_MSG(requesting, ERR_BUSY, "HTTPRequest is processing a request. Wait for completion or cancel it before attempting a new one.");
 
-	if (timeout > 0) {
+	if (timeout > CMP_EPSILON) {
```

Comment:
```suggestion
	if (!Math::is_zero_approx(timeout)) {
```

Reply:
Actually... we can likely leave `timeout > 0.0` here, since we force-set that value, and constant values should be consistent AFAIK (i.e. `0.0 > 0.0 == false`), and any other allowed value would be `> 0.0`.